Demo/Tweakable/TweakableDemoView.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import FinniversKit
+
+public class TweakableDemoView: UIView, Tweakable {
+    lazy var tweakingOptions: [TweakingOption] = {
+        var options = [TweakingOption]()
+
+        options.append(TweakingOption(title: "Option 1", description: nil, action: { parentViewController in
+            self.titleLabel.text = "Choosen Option 1!\n\nYou can drag the button too :D"
+            parentViewController?.dismiss(animated: true)
```

Comment:
Shouldn't it be the responsibility of `TweakingOptionsTableViewController` to dismiss the presented viewController/bottomSheet?